nio_cli/commands/config.py

```diff
@@ -11,8 +12,11 @@ def config_project(name='.'):
         print("Command must be run from project root.")
         return
 
-    pk_host = input('Enter Pubkeeper hostname (optional): ')
-    pk_token = input('Enter Pubkeeper token (optional): ')
+    print('')
+    print('')
+    print('\033[92m' + 'Configure your local instance:' + '\033[0m')
+    pk_host = input('Enter Pubkeeper hostname (required): ')
```

Comment:
Required! Eeek! What if I want to connect to the local pubkeeper server in the nio instance?

Reply:
I mean, right now, it's required... if you don't put in a value, nio doesn't start... I'd LOVE to not have it be required, but it used to say (optional), and that's just not true.

Reply:
It's not required if you start nio with an embedded pubkeeper server
```
niod -s nio.conf -s pk_server.conf
```

But maybe that could be more clear in the docs?

Reply:
What are the values that go into the nio.conf file for a standalone pubkeeper server? 127.0.0.1 and whatever the token is they put in pk_server.conf?

Reply:
the cli now accommodates and configures the internal pk server, including adding the niod -s instructions on completion